src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const item of transforms) {
```

Comment:
```suggestion
  for (const { operation, properties } of transforms) {
```

Можна зробити ось таку деструктуризацію, тоді нижче можна буде не писати item.properties, а просто properties